indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -64,17 +64,17 @@
     background-color: lighten($dark-gray, $color-variation);
     cursor: move;
 
-    &:before,
-    & > .handle:before {
+    &::before,
+    & > .handle::before {
```

Comment:
Actually, it would be nice to make sass-lint warn against this. But I can't find a rule for it.